crates/sncast/tests/e2e/account/create.rs

```diff
@@ -47,10 +47,12 @@ pub async fn test_happy_case(account_type: &str) {
     assert_stdout_contains(
         output,
         indoc! {r"
-        command: account create
-        address: 0x0[..]
-        estimated_fee: [..]
-        message: Account successfully created but it needs to be deployed. The estimated deployment fee is [..]
+        Success: Account created
+
+        Address:       0x0[..]
+        Estimated Fee: [..]
```

Comment:
Lack of padding in `Success: Account created` line feels a bit weird. How about some kind of divider after the title, e.g.

```
        Account created!
        --------------------
        Address:       0x0[..]
        Estimated Fee: [..]
```

wdyt?